demo-linux/demo.cpp

```diff
@@ -67,6 +71,26 @@ void loadEEPROM()
     free(values);
 }
 
+int writeToEEPROM()
+{
+    unsigned char buf[64];
+    string writeString;
+    wp_get_eeprom_page(specIndex, 4, buf, 64);
+    printf("The value at page 4 is %s\n", buf);
+    std::stringstream s;
+    s << "Test string. ";
+    s << time(NULL);
+    writeString = s.str();
+    strcpy((char*)buf, writeString.c_str());
```

Comment:
I'm sure this works, and it's not likely to overrun, but ANY call to strcpy() is dangerous.  strncpy() is always preferred :-)